dyn_em/module_first_rk_step_part2.F

```diff
@@ -254,6 +255,35 @@ SUBROUTINE first_rk_step_part2 (   grid , config_flags        &
                            grid%stddev_cutoff_rand_pert,grid%gridpt_stddev_rand_pert, & 
                            grid%VERTSTRUCC,grid%VERTSTRUCS,grid%VERTAMPT       )
       ENDIF !rand_perturb_on
+
+      If_multi_perturb: if ((config_flags%multi_perturb==1).and.(grid%id == 1) .and. &
+          ((MOD(grid%itimestep, grid%stepsp) == 0) .or. grid%itimestep == 1)) then
```

Comment:
@pedro-jm @dudhia 
Here's the trouble.
When stepsp is zero, we do mod(something,0), and that is a floating point error. There needs to be some extra IF tests about stepsp .ne. 0